recipes/xapian-core/all/conanfile.py

```diff
@@ -59,91 +60,103 @@ def requirements(self):
 
     def validate(self):
         if self.options.shared and self.settings.os == "Windows":
-            raise ConanInvalidConfiguration("shared builds are unavailable due to libtool's inability to create shared libraries")
+            raise ConanInvalidConfiguration(
+                "shared builds are unavailable due to libtool's inability to create shared libraries"
```

Comment:
Please change these to be more positive /encouraging for people to contribute 🙏

Reply:
I didn't change this message, it's out of scope of this PR.